meson_options.txt

```diff
@@ -14,8 +14,8 @@ option('openblas_include',
        description: 'Paths to openblas include directories')
 
 option('opencl_include',
-       type: 'array',
-       value: ['/usr/include/'],
+       type: 'string',
+       value: '/usr/include/',
        description: 'Paths to OpenCL include directories')
```

Comment:
This is really nitpicking, but now it's "Path" :)

Reply:
If only every review comment was this easy..

Reply:
...and if only I was more careful when writing patches!